test/verify/check-machines-dbus

```diff
@@ -1960,7 +1968,7 @@ class TestMachinesDBus(machineslib.TestMachines):
                 self.test_obj.assertEqual(size, int(self.size))
 
             def cleanup(self):
-                m.execute("virsh pool-destroy {0} && virsh pool-undefine {0}".format(self.name))
+                m.execute("virsh pool-destroy {0} && virsh pool-undefine {0}".format(self.pool_name))
```

Comment:
This is correct fix but should be in separate commit since it has nothing to do with the new test.
When you repost with the new commit don't forget to add a Closes #13927 line in your commit message.

Reply:
OK, many thanks